src/analytics/index.ts

```diff
@@ -0,0 +1,3 @@
+import * as actions from "./actions"
```

Comment:
This will soon also house `events` so that we can have type safety around what event strings can be sent. For now, this module is pretty empty, but separating it out makes it easier to check that it's being called as expected in the tests.